starter-code/src/functions-and-arrays.js

```diff
@@ -11,14 +21,43 @@ var words = [
   'crackpot'
 ];
 
+
+function findLongestWord (words) {
+  var maxLengthWord = words[0].length;
```

Comment:
You should assign an empty value to this variable, per exempl : 

```js

var maxLengthWord = ""

```

It's better, because words (your array) can be empty, so you won't be able to refer to the index 0 of the Array, if you use this, it will remove some error from jasmine :)